specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -1237,12 +1237,28 @@
           "description": "Location of the backend (IP address or FQDN)",
           "type": "string"
         },
-        "privateLinkId" : {
-          "description": "If this backend is private, provide the private link resource Id. Populating this optional field indicates that this backend is 'Private'",
+        "privateLinkResourceId": {
+          "description": "If this backend is private, provide the Private Link resource Id. Populating this optional field indicates that this backend is 'Private'",
           "type": "string"
         },
-        "location": {
```

Comment:
Same comment as above, why the nreaking change?

Reply:
Please see answer above